app/models/training/content.rb

```diff
@@ -16,6 +16,16 @@ def parent
       @parent ||= Training::Module.by_content_id(id)
     end
 
+    # @return [Boolean]
+    def updated_content?
+      created_at < updated_at
+    end
+
+    # @return [Date]
+    def published_at_date
```

Comment:
`published_at` would be sufficient because that is a convention that implicitly tells us we will receive a `DateTime`. Also, is the yard return value accurate?